src/main.cpp

```diff
@@ -44,60 +45,44 @@ void setup()
 
     // Motor UART init
     motor_serial.begin(19200);
+    motor_serial.setTimeout(50);
```

Comment:
Is this based on something or just a random timeout?

Reply:
a bit more than twice the expected time (23.44ms). I like round numbers